### DISCUSSION TITLE
Photogrammetry Example?

### DISCUSSION THREAD

**Discussion by stanleyshly** [dfki-ric/pytransform3d#179] [Q&A] [answered]
In the Readme, there is an example of a tree trunk with the computed position of the camera being displayed with the reconstructed 3D mesh. Does an example do this at the current moment?

**Comment by AlexanderFabisch** [comment, marked as answer]
I did not include the script in the repository since it requires the mesh, camera configuration and poses additionally. The script that I used to produce the image can be found here: https://github.com/rock-learning/pytransform3d/discussions/148 . The configuration files that I load here were produced by Meshroom.
